spec/models/collections_spec.rb

```diff
@@ -84,4 +84,30 @@
       expect(collection).to_not be_published
     end
   end
+
+  describe "validate uid does not change after create" do
+    let(:vendor) { Vendor.create!(uid: 'voice_base', name: 'VoiceBase') }
+    let(:collection) do
+      Collection.new(
+        uid: "collection_test",
+        title: "test collection",
+        url: "https://catalogue_collection",
+        description: "test collection",
+        vendor_id: vendor.id
+      )
+    end
+
+    context "when a new collection is saved" do
```

Comment:
ie, we're testing validations. so best to check whether the record is valid or not (and any error messages) rather than just whether `update` returned `false`